Automattic/studio
# Studio: Update wordpress/components and wordpress/icons dependencies

Related to https://github.com/Automattic/dotcom-forge/issues/8980

## Proposed Changes

In https://github.com/Automattic/dotcom-forge/issues/8980 , we need to use new icons for the chat rating feature `thumbsUp` and `thumbsDown`. This PR updates dependencies for `wordpress/components` and `wordpress/icons` so that we can use the new set of icons that were added last week.

## Testing Instructions

* Pull the changes from this branch
* Start the app with `nvm use && npm install && npm start`
* Observe that the icons work as expected e.g. they display (you can check `chevronRight` or `chevronDown` icons on the `Add site` form)
* Holistically check that the app works
* Observe no errors in the console

## Pre-merge Checklist
